bofire/data_models/kernels/kernel.py

```diff
@@ -1,5 +1,15 @@
+from typing import List, Optional
+
 from bofire.data_models.base import BaseModel
 
 
 class Kernel(BaseModel):
     type: str
+
+
+class AggregationKernel(Kernel):
+    pass
+
+
+class ConcreteKernel(Kernel):
```

Comment:
Should we give it a more verbose name like `FeatureSpecificKernel`? But ConcreteKernel is also fine for me ;)